src/migration.ts

```diff
@@ -205,10 +205,12 @@ export async function compileIncludes(
   // Simple string replacement for each path matched
   const compiledContent = content.replace(
     regex,
-    (_match, rawSqlPath: string) => {
+    (match, rawSqlPath: string) => {
       const sqlPath = sqlPathByRawSqlPath[rawSqlPath];
       const content = contentBySqlPath[sqlPath];
-      return content;
+      const included = match.replace(/^--![ \t]*include/, "--! Included");
```

Comment:
If you make the change above to allow `Include` as well as `include`, this will break. I don't think we want the change on line 135. I feel like we should check for no match, but meh.